data.json

```diff
@@ -8,6 +8,15 @@
     "dateAdded": "2018-11-23T17:27:00.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "Accord Project",
+    "difficulty": "hard",
+    "description": "Have two accepted pull request in any Accord Project will receive the sticker. Have  1 valid ‘Difficulty: Challenging‘ PR (OR) 2 valid ‘Difficulty: Medium‘ PRs (OR) 5 valid ‘Difficulty: Starter‘ PRs will get the T-shirt.",
```

Comment:
This seems a bit complicated. I can't think of the best wording right now, so I'll let the other reviewers make suggestions 😄